src/v/cluster/scheduling/leader_balancer.cc

```diff
@@ -298,6 +298,13 @@ ss::future<ss::stop_iteration> leader_balancer::balance() {
         co_return ss::stop_iteration::no;
     }
 
+    vlog(
+      clusterlog.info,
+      "Transferred leadership for group {} from {} to {}",
+      transfer->group,
+      transfer->from,
+      transfer->to);
+
```

Comment:
is something like this already logged near the bottom of this routine?

Reply:
Thanks. Not sure how I missed that.